<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevEdit</class>
 <widget class="QDialog" name="DevEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>851</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_57" stretch="5,4,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QGroupBox" name="groupBox_25">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(238, 238, 236);</string>
        </property>
        <property name="title">
         <string>通信接口配置</string>
        </property>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_33" stretch="2,5">
          <item>
           <widget class="QLabel" name="label_49">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(239, 41, 41);
background-color: rgb(211, 215, 207);</string>
            </property>
            <property name="text">
             <string>接口类型：</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QRadioButton" name="radioButton_Com">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(173, 127, 168);</string>
              </property>
              <property name="text">
               <string>串口设备</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Net">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(173, 127, 168);</string>
              </property>
              <property name="text">
               <string>网口设备</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>100</y>
           <width>301</width>
           <height>261</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string/>
          </attribute>
          <widget class="QWidget" name="widget_port" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>281</width>
             <height>201</height>
            </rect>
           </property>
           <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>261</width>
              <height>178</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1,1,1">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,4">
               <item>
                <widget class="QLabel" name="label_52">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>端口号：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_6">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <item>
                  <property name="text">
                   <string>COM1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM4</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM5</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM6</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM7</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM8</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM9</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM10</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM11</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM12</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>USB-COM1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>USB-COM2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>USB-COM3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>USB-COM4</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>XR-COM1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>XR-COM2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>XR-COM3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>XR-COM4</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,4">
               <item>
                <widget class="QLabel" name="label_55">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>波特率：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_devBaud">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <item>
                  <property name="text">
                   <string>600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4800</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>19200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>38400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>115200</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,4">
               <item>
                <widget class="QLabel" name="label_56">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>停止位：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_devStopBit">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="1,4">
               <item>
                <widget class="QLabel" name="label_60">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>校验位：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_devParity">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <item>
                  <property name="text">
                   <string>N</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>E</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>O</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,4">
               <item>
                <widget class="QLabel" name="label_57">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>数据位：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_devDatabit">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <item>
                  <property name="text">
                   <string>7</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string/>
          </attribute>
          <widget class="QTextEdit" name="textEdit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>90</y>
             <width>231</width>
             <height>81</height>
            </rect>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ef2929;&quot;&gt;注：服务地址为空，则为串口设备，否则为网口设备。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_ipAddr">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>50</y>
             <width>238</width>
             <height>25</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(211, 215, 207);</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_58">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>81</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>服务地址：</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="title">
       <string>设备基本配置</string>
      </property>
      <widget class="QWidget" name="widget_basicDevSet" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>261</width>
         <height>301</height>
        </rect>
       </property>
       <widget class="QWidget" name="layoutWidget_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>251</width>
          <height>148</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_59">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>设备ID：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_id">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_32" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_54">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>设备地址：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_devAddr">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_31" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_53">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>设备类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_devProto">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_36" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_devName">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>设备名称：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Name">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(211, 215, 207);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_Detail">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>设备类型说明</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="title">
       <string>设备参数</string>
      </property>
      <widget class="QTextEdit" name="textEdit_devParams">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>191</width>
         <height>351</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(211, 215, 207);</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_UpdateDev">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>500</y>
     <width>131</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>131</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
